src/assets/translations/de.json

```diff
@@ -0,0 +1,34 @@
+{
+    "@metadata": {
+      "authors": [
+        "Michael Fuchs"
+      ],
+      "last-updated": "2023-07-05",
+      "locale": "de"
+    },
+    "cta-button-placeholder": "Zugriff über Ihre Institution",
+    "cta-button-header": "Zugriff über",
+    "cta-link": "Institution hinzufügen oder ändern",
+    "ds-header": "Zugriff auf",
+    "ds-footer-about-us": "Über uns",
+    "ds-search-heading": "Finden Sie Ihre Institution",
+    "ds-search-subheading": "Ihre Universität, Organisation oder Firma",
+    "ds-search-example": "Beispiele: Science Institute, Lee@uni.edu, UCLA",
```

Comment:
would it make sense to have examples that resonate with German-speaking users instead?

Reply:
Yea, that makes sense. I've updated the PR.